packages/InteractiveProfilingTool-Core.package/IPTTreeModel.class/instance/windowTitle.st

```diff
@@ -0,0 +1,3 @@
+code browser
+windowTitle
+	^ 'Interactive Profiler'
```

Comment:
Please consider adding a newline after the function name to be consistent with other methods.